input/pagecontent/changes.md

```diff
@@ -188,3 +189,4 @@ This change log documents the significant updates and resolutions implemented fr
   - replaced security and authorization requirements with pointer to Security and Privacy page [FHIR-46067](https://jira.hl7.org/browse/FHIR-46067)
   - corrected narrative of PractitionerRole practitioner.identifier search parameter from Medicare Provider Number to HPI-I [FHIR-47013](https://jira.hl7.org/browse/FHIR-47013)
   - corrected support for Practitioner _id search parameter to be SHOULD instead of SHALL [FHIR-46777](https://jira.hl7.org/browse/FHIR-46777)
+  - added entries for referenced resources, DocumentReference and RelatedPerson, in line with the Must Support requirements in AU Core profiles [FHIR-46546](https://jira.hl7.org/browse/FHIR-46546)
```

Comment:
rephrase to just "added missing entry for DocumentReference and RelatedPerson resources."
this text "in line with ..." is like gravity - it applies to all changes ever.

Reply:
Amended, thanks

Reply:
spelling issues. entry should be plural.

Reply:
Amended, thanks